.github/workflows/dockerimage.yml

```diff
@@ -46,6 +47,10 @@ jobs:
       run: |
         docker build . --build-arg PYTHON_VERSION=${{ matrix.python_version }} --build-arg BUILD_TYPE=${{ matrix.build_type }} --file Dockerfile --tag "${HUB_TAG}"
 
+    - name: Run an example using the built image
+      run: |
+        docker run "${HUB_TAG}" python examples/sklearn_simple.py
```

Comment:
How about checking whether libraries are correctly installed or not by checking `pip freeze` instead of running a concreate example as running example might be time consuming?

Reply:
How about `tutorial/001_first.py` which should run quick?

Reply:
> How about tutorial/001_first.py which should run quick?

This approach seems reasonable to me. I think we may overlook runtime error if we only check with `pip freeze`.

Reply:
Agree with @toshihikoyanase. @crcrpar What do you think?

Reply:
Personally https://github.com/optuna/optuna/blob/master/examples/quadratic_simple.py looks better than `tutorial/001_first.py` :smiley:

Reply:
@crcrpar `quadratic_simple.py` looks good.